CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 19/02/2020
+- Add Areas v2 router, which syncs areas of interest with subscriptions from the GFW Subscriptions API MS. For more information about this feature, check this document: https://docs.google.com/document/d/13I-kGmkoxeZF3l4yZub2p1jrpLpGm25sC0lcCQyFNcI/edit?usp=sharing
```

Comment:
If the content of this doc should be public, it may make sense to port that info either to the docs, to the readme, or another .md in this repo.

Reply:
The most important parts of that doc were added to the docs in this PR: https://github.com/resource-watch/doc-api/pull/114

Going to remove the link to the doc here and mention to check the docs.